### DISCUSSION TITLE
Make Settings menu use the whole screen instead of 1/3's of it

### DISCUSSION THREAD

**Discussion by Skubers** [ppy/osu#25896] [Ideas]
Since there are -so many settings- available on lazer, there's no reason to make it an infinitely scrollable list of settings, especially when it's hard to distinguish between different sections of the Settings because the font and its size used for category name and setting name is very similar 
![image](https://github.com/ppy/osu/assets/82095491/08e98c86-54d7-42ee-8928-a34c143520f6)

I doubt users need to see the osu! cookie, or the song select when changing settings, so best option would be to just make it utilize the whole 16:9/21:9/32:9 monitor, so that it's not an infinitely scrollable list.

If you want to keep it for mobile's sake, you can make so that first click of Ctrl+O opens settings as they are now, but clicking Ctrl+O (or an expand button on Settings popup) will bring out the full screen Settings page, where everything will be much easier to manage, distinguish, etc. Each category would easily fit its settings into a non-scrollable screen of settings.

**Comment by bdach** [comment]
> Since there are -so many settings- available on lazer

i personally consider this to be the primary problem

some of these will get moved around to more appropriate places (e.g. skin editor) as time moves on. the other thing is that having this be a sidebar allows adjusting settings from pretty much any screen.

as for spacing / typography issues... @arflyte

**Reply by Skubers** [reply]
That's why I suggested that Settings could have two states, the sidebar state and the expanded state.

I'm not talking about a whole different section of the game designed to be a Settings page, but rather a full-screen pop-up with settings, kinda like the Setup Wizard

**Comment by peppy** [comment]
Making it fullscreen along won't help. The issue is it's just a single long list, so it's hard to learn where things are. It's why other apps make grids etc. to make the layout memorable.

We'll definitely improve this in the future.
